azext_hanaonazure/azext_metadata.json

```diff
@@ -1,4 +1,4 @@
 {
   "azext.minCliCoreVersion": "2.0.46",
-  "version": "0.3.9"
```

Comment:
Is this a generated file as well? Why does this repo have versions in two places?

Reply:
Not generated, and out of the scope of this PR for why there are versions in several places. I agree it is confusing, but it isn't something I'm going to address at the moment.